x-pack/platform/plugins/shared/cases/public/components/actions/use_items_action.tsx

```diff
@@ -33,7 +33,7 @@ export const useItemsAction = <T,>({
   const { permissions } = useCasesContext();
   const [isFlyoutOpen, setIsFlyoutOpen] = useState<boolean>(false);
   const [selectedCasesToEdit, setSelectedCasesToEdit] = useState<CasesUI>([]);
-  const canUpdateStatus = permissions.update;
+  const canUpdateStatus = permissions.update || permissions.assign;
```

Comment:
would it be better to return a bool specific to assignment as it looks like `isActionDisabled` also affects other actions, that should still be available without assign privs?